pkg/controller/infinispan/resources/operatorconfig/monitoring_constants.go

```diff
@@ -1200,47 +1200,47 @@ const dashboardJSON = `{
 			},
 			"targets": [
 			  {
-				"expr": "max({__name__=~\"vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries\"})",
+				"expr": "max({namespace=~\"$namespace\", job=~\"$cluster\", __name__=~\"vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries\"})",
 				"format": "time_series",
 				"instant": false,
 				"intervalFactor": 1,
 				"legendFormat": "Entries",
 				"refId": "A"
 			  },
 			  {
-				"expr": "max({__name__=~\"vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries_in_memory\"})",
+				"expr": "max({namespace=~\"$namespace\", job=~\"$cluster\", __name__=~\"vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries_in_memory\"})",
```

Comment:
Btw this one is now shown in the visualization at all. I think the metric is incorrect (missing `current` word):
Is: `vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_number_of_entries_in_memory`
Should be: `vendor_cache_manager_default_cache_($caches)_cluster_cache_stats_current_number_of_entries_in_memory`

Reply:
Ah, yeah the stat name changed at some point :(